service/controller/resource/deployment/templates.go

```diff
@@ -0,0 +1,1523 @@
+package deployment
```

Comment:
There is no built-in way in Go to read `resources` in the same package. So I decided to inline the templates using constants.